### ISSUE TITLE
Setup not working - Fix xComputerManagement required version

### ISSUE THREAD

**Issue by alexjriggs** [pluralsight/PS-AutoLab-Env#196]
Probably doing something really simple wrong but it’s eluding me. I’ve read all the instructions and postings.
Started with a fresh install of Windows and PS-Autolab
PSVersion : 5.1.17763.592
Edition   : Desktop
OS        : Microsoft Windows 10 Enterprise LTSC
PSAutolab : 4.2.0
Lability  : 0.18.0
Memory    : 4100236
Followed all the steps in the Readme.md file but Unattend-Lab will not download anything so I manually downloaded the ISO files and put them in the Autolab\ISOs directory. Using Get-Labmedia cmdlet from the Lability module to find the download link and saved the file using the filename.
Still none of the labs will configure. Here’s what happens.
PS C:\Autolab\Configurations\PowerShellLab> Unattend-Lab
       This runs Setup-Lab, Run-Lab, and Validate-Lab
Starting the lab environment
        This is the Setup-Lab script. This script will perform the following:

        * Run the configs to generate the required .mof files
Note! - If there is an error creating the .mofs, the setup will fail
        * Run the lab setup
Note! If this is the first time you have run this, it can take several        hours to download the ISO files and resources depending on the configuration. You may also see new drives being added and removed as the ISO is mounted and converted. This step should only happen the first time you run this command.
        ** Possible problem
If the downloads finish but the script doesn't continue (pauses), press the Enter key once and it should continue
*You will be able to wipe and rebuild this lab without needing to perform the downloads again. 
Installing required DSCResource modules from PSGallery 
You may need to say "yes" to a Nuget Provider
xActiveDirectory [v3.0.0.0] requires no updates.
xComputerManagement [v4.1.0.0] requires no updates.
xNetworking [v5.7.0.0] requires no updates.
xDhcpServer [v2.0.0.0] requires no updates.
xWindowsUpdate [v2.8.0.0] requires no updates.
install xPSDesiredStateConfiguration version 8.9.0.0
xPendingReboot [v0.4.0.0] requires no updates.
xAdcsDeployment [v1.4.0.0] requires no updates.
xDnsServer [v1.14.0.0] requires no updates.
xWebAdministration [v2.7.0.0] requires no updates.
Build the .Mof files from the configs
At C:\Autolab\Configurations\PowerShellLab\VMConfiguration.ps1:26 char:5
+     Import-DSCresource -ModuleName PSDesiredStateConfiguration,
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Could not find the module '<xComputerManagement, 2.0.0.0>'.
    + CategoryInfo          : ParserError: (:) [], ParseException
    + FullyQualifiedErrorId : ModuleNotFoundDuringParse

Building the lab environment for PowerShellLab
Failed to start lab configuration.
Cannot locate node 'DOM1' file and no DSC configuration can be applied. Ensure the DSC configuration has been run successfully.
At line:114 char:13
+             throw ($localized.CannotLocateMofFileError -f $Name);
+             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (Cannot locate n...n successfully.:String) [], RuntimeException
    + FullyQualifiedErrorId : Cannot locate node 'DOM1' file and no DSC configuration can be applied. Ensure the DSC configuration has been run successfully.

Any help would be greatly appreciated. 
Thank you,

**Comment by jdhitsolutions** [comment]
I had found an issue with DSC resource versioning that I thought I had fixed. In the meantime, I think the short term fix is to manually install the resource setup is looking for.  In an elevated PowerShell session run:

```powershell
Install-Module xComputerManagement -requiredversion 2.0.0.0 -force
```

Then try setup again.

**Comment by alexjriggs** [comment]
Thank you so much. It is downloading and trying to setup the lab. Unfortunately, I'm getting this error for the last several hours now:

Invoke-Pester : A parameter cannot be found that matches parameter name 'Show'.
At C:\Program Files\WindowsPowerShell\Modules\PSAutoLab\4.2.0\functions\public.ps1:478 char:49
+         $test = Invoke-Pester -Script $testpath -Show none -PassThru
+                                                 ~~~~~
    + CategoryInfo          : InvalidArgument: (:) [Invoke-Pester], ParameterBindingException
    + FullyQualifiedErrorId : NamedParameterNotFound,Invoke-Pester

**Comment by alexjriggs** [comment]
Sorry I just read problem #195 and it applies to my setup. I'll try it that. :)

**Comment by jdhitsolutions** [comment]
This should be addressed in v4.3.0 which updates Pester during host setup.
